app/assets/stylesheets/index.css.scss

```diff
@@ -70,6 +70,14 @@ body[data-controller="home"][data-action="index"] {
     img { display: none; }
     .mission_text { font-size: 1.7em; }
   }
+  #coder_welcome {
```

Comment:
Avoid using id selectors<br>Selector `coder_welcome` should be written in lowercase with hyphens